synapse/rest/client/v2_alpha/report_event.py

```diff
@@ -31,12 +40,26 @@
 class ReportEventRestServlet(RestServlet):
     PATTERNS = client_patterns("/rooms/(?P<room_id>[^/]*)/report/(?P<event_id>[^/]*)$")
 
+    hs: HomeServer
+    auth: Auth
+    federation_client: FederationClient
+    snm: ServerNoticesManager
+    state: StateHandler
+    storage: Storage
+    store: DataStore
+
     def __init__(self, hs):
         super().__init__()
         self.hs = hs
         self.auth = hs.get_auth()
+
+
         self.clock = hs.get_clock()
+        self.federation_client = hs.get_federation_client()
+        self.snm = hs.get_server_notices_manager()
```

Comment:
Please use a longer variable name.

Reply:
Ok, I've been using what I found around the Synapse source code, but I'll adapt :)